static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -273,7 +275,7 @@ define([
                                             }
                                         });
                                         player.on('adstart', function() {
-                                            player.adSkipCountdown(15);
+                                            player.skipAd(mediaType, 15);
```

Comment:
It fires and event saying it's been skipped.
The event name, for now, are `media:{mediaType}:{eventType}`.